src/utils/periods/__tests__/relativePeriods.js

```diff
@@ -0,0 +1,139 @@
+import { getRelativePeriodStartDate } from '../relativePeriods.js'
+
+describe('getRelativePeriodStartDate', () => {
+    const originalDateNow = Date.now
+
+    beforeAll(() => {
+        global.Date.now = jest.fn(() =>
```

Comment:
Jest has a nice way of mocking globals with [spyOn](https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname), it requires a little less manual effort. For example:

```js
jest.spyOn(global.Date, 'now').mockImplementation(() => 'someDate')
```

Afterwards you can clean it up with [mockRestore](https://jestjs.io/docs/en/mock-function-api#mockfnmockrestore). (I'm not sure if the jest version you have here supports it though)

Reply:
Addressed in 3ecb1d7